express-app/controllers/estudio.js

```diff
@@ -0,0 +1,176 @@
+const Estudio = require('../models/estudio');
+const Familia = require('../models/familia');
+const familyController = require('./family');
+module.exports = {
+  /**
+  * This function shows the family form, IF the url query is empty
+  * then it means that the familia and estudio is created from scratch, IF NOT then
+  * it means it is a estudio that is going to be updated.
+  *
+  * @event
+  * @param {object} request - request object 
+  * @param {object} response - response object.
+  */
+  showFamilyForm: function(request, response) {
+    //retrieve estudio id from url
+    let estudioId = request.query.estudioId;
+    if (estudioId) {
+      Estudio.findOne({
+        _id: estudioId
+      })
+      .then((myEstudio) => {
+        response.render('family',  {
+          estudioId: myEstudio._id, 
+          family: myEstudio.familia
+        });
+      })
+      .catch((error) => {
+        //no estudio found
+        console.log(error);
+      })
+    }
+    else {
+      response.render('family');
+    }
+  },  
+  /**
+  * This function creates a Estudio with te apiToken of the capturista,
+  * and with a family,
+  * then it shows the form to fill the members information.
+  * 
+  * 
+  * @event
+  * @param {object} request - request object 
+  * @param {object} response - response object.
+  */    
+  createEstudio: function(request, response) {
+    //retrieve token from params
+    const token = request.session.user.apiToken;
+    //get data from request
+    const data = request.body;
```

Comment:
no, all validation occurs at front, do you think is still necessary to sanitize data? @erikiado